core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -181,7 +181,7 @@ export class QuestionBackendApiService {
    * Returns a list of questions based on the list of skill ids and number
    * of questions requested.
    */
-  fetchQuestions(
+  async fetchQuestionsAsync(
```

Comment:
Should you make changes in the place where this is used too?

Reply:
Please check that the methods that you have modified have been changed in all the places they are used.

Reply:
> Please check that the methods that you have modified have been changed in all the places they are used.

done! 
I have checked and made changes to the function names wherever 
`core/templates/domain/question/question-backend-api.service.ts`  is imported for class QuestionBackendApiService.

Reply:
Hi @MegaMind77-coder, I think that does not cover all the cases. We "require" this service in some files so searching for "import ..." won't be sufficient. For example, you have missed a case in `core/templates/pages/exploration-player-page/services/exploration-player-state.service.ts`.

I would do a search for all occurrences of the function name, then for each usage, I would check if the method belongs to the correct service and (only then) rename it. Please let me know if you have any questions.

Reply:
Hi @kevintab95 , thanks for pointing it out. I have made the required change in  
`core/templates/pages/exploration-player-page/services/exploration-player-state.service.ts`. I had also missed cases in `core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.ts` and `core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.spec.ts`, I have now made the required changes. 
Thanks